projects/kubernetes-csi/external-provisioner/ATTRIBUTION.txt

```diff
@@ -53,10 +53,10 @@ https://github.com/grpc/grpc-go
 ** gopkg.in/yaml.v2; version v2.4.0 --
 https://gopkg.in/yaml.v2
 
-** k8s.io/api; version v0.21.0-alpha.3 --
+** k8s.io/api; version v0.20.4 --
```

Comment:
These changes come from the change to the generator to properly handle replace blocks in go.mod files